spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -854,12 +867,14 @@ public void testCompareToFileList() throws IOException {
         .as("Invalid file should be present")
         .isTrue();
 
-    DeleteOrphanFiles.Result result3 =
+    DeleteOrphanFilesSparkAction action3 =
```

Comment:
While we are modifying things here, please rename all name# variables to something relevant to the test. The names should be relevant to what we are checking

Reply:
renamed to more descriptive names